doorbot.py

```diff
@@ -57,11 +55,7 @@
 GPIO_EVENT_DOUBLECHECK_DELAY = 0.03  # s
 BUSY_WAIT_SLEEP_DURATION = 0.05  # s
 
-DOOR_OPENING_TIME_RANGE = {
-    "downstairs_immediate": ("mon-fri", "08:00", "19:00"),
-    "downstairs_delayed": ("sat-sun", "09:00", "18:00"),
-    "office": ("mon-fri", "08:45", "19:00"),
-}
+assert DOORBELL_MODE in ("chirp", "doorphone")
```

Comment:
chirp :smile:

Reply:
In the sense of "bird sound", not "exponential frequency sweep" :)